src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  font-family: Roboto, sans-serif;
+  min-width: 1023px;
+  max-width: 1201px;
+  height: 60px;
+}
+
+:root {
+  --color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__link {
+  font-size: 12px;
+  text-decoration: none;
+  color: #000;
+  padding-top: 22px;
+  padding-bottom: 22px;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  flex-wrap: nowrap;
+}
+
+.nav__item {
+  font-weight: 600;
+  text-transform: uppercase;
+  margin-right: 20px;
+}
+
+.nav {
+  padding-right: 30px;
+}
+
+.img {
+  height: 40px;
+  width: 40px;
+  margin-top: 10px;
+  margin-left: 50px;
+}
+
+.is-active {
+  position: sticky;
```

Comment:
The `position: sticky;` property in the `.is-active` class might not be necessary or appropriate here. Ensure that this property is used intentionally, as it can affect the layout in unexpected ways.